nekoyume/Assets/_Scripts/UI/Widget/Screen/IntroScreen.cs

```diff
@@ -274,21 +274,39 @@ private IEnumerator CoSocialLogin()
             yield return new WaitForSeconds(1);
             Game.Game.instance.PortalConnect.OpenPortal(() => popup.Close());
         }
-        
-        private void ApplyCurrentPlanetInfo(PlanetInfo planetInfo)
+
+        private void ApplyCurrentPlanetInfo(PlanetContext planetContext)
         {
-            if (planetInfo is null)
+            var planets = planetContext.Planets;
+            var planetInfo = planetContext.CurrentPlanetInfo;
+            if (planets is null ||
+                planetInfo is null)
             {
                 planetText.text = "Null";
                 heimdallButton.Interactable = false;
```

Comment:
It isn't the scope of this PR but... I think that we should avoid to use planet-name specific codes as possible since planets can be changed separately from codebase.

Reply:
I agree, and I have not enough time to consider and develop here. It's actually error case in full context and.. I'll handle it next week!